docs/conf.py

```diff
@@ -5,6 +5,9 @@
 import os
 import shlex
 
+# Add the folder where autoprocess.py extension is located to python path, so sphinx can locate it:
+sys.path.append(os.path.abspath(os.path.dirname(__file__)))
```

Comment:
Save `os.path.abspath(os.path.dirname(__file__))` to variable and use here and in `base_dir = ...`